src/tox/_pytestplugin.py

```diff
@@ -176,9 +176,12 @@ def outlines(self):
         return err + out
 
     def __repr__(self):
-        return "RunResult(ret={}, args={}, out=\n{}\n, err=\n{})".format(
+        res = "RunResult(ret={}, args={}, out=\n{}\n, err=\n{})".format(
             self.ret, " ".join(str(i) for i in self.args), self.out, self.err
```

Comment:
this will error on non-ASCII unicode args in py2, in my branch I Changed this to `args={!r}` (which is more readable anyway